runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettingsInternal.java

```diff
@@ -95,6 +106,8 @@
     private boolean mSpatialNavigationEnabled = true;
     private boolean mQuirksModeEnabled = false;
 
+    private LayoutAlgorithmInternal mLayoutAlgorithm = LayoutAlgorithmInternal.NARROW_COLUMNS;
```

Comment:
@wuhengzhi Are you positive this default is correct? Android's WebView says that NORMAL is the recommended mode, NARROW_COLUMS is deprecated, and I didn't change the mode in my test project.